libs/common/setup.go

```diff
@@ -23,9 +23,11 @@ var (
 const DevelopmentMode = "development"
 const ProductionMode = "production"
 
+var skipAuthForMethods []string
```

Comment:
The naming could be better or more related to the parameter passed in `Setup()`.